ncm-metaconfig/src/main/pan/components/metaconfig/config-common.pan

```diff
@@ -1,17 +0,0 @@
-# ${license-info}
-# ${developer-info}
-# ${author-info}
-# ${build-info}
-
-unique template components/${project.artifactId}/config-common;
-
-include { 'components/${project.artifactId}/schema' };
-
-# Set prefix to root of component configuration.
-prefix '/software/components/${project.artifactId}';
-
-#'version' = '${project.version}';
-#'package' = 'NCM::Component';
```

Comment:
i would like to start using the version again for https://github.com/quattor/ncm-ncd/issues/41, but this PR just cleans up teh `config-*.pan`.
i'm no sure i understand what you mean with `functionality to register the /software/packages/ncm-$component/ subtree`. i thought that was the default for each component? but i could be wrong....